file-formats/edck/type/zif_aff_edck_v1.intf.abap

```diff
@@ -0,0 +1,74 @@
+INTERFACE zif_aff_edck_v1
+  PUBLIC.
+
+  TYPES ty_check_classification TYPE c LENGTH 1.
+  TYPES ty_check_category TYPE c LENGTH 6.
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Check Category</p>
+      "! Consistency Check Category
+      "! $values {@link zif_aff_edck_v1.data:co_check_category}
+      "! $default {@link if_aff_edck_v1.data:co_check_category.exist}
+      "! $required
+      checkcategory             TYPE ty_check_category,
+      "! <p class="shorttext">Check Classification</p>
+      "! Consistency Check ID Classification
+      "! $values {@link zif_aff_edck_v1.data:co_check_classification}
+      "! $default {@link if_aff_edck_v1.data:co_check_classification.core}
+      "! $required
+      checkidclassification     TYPE ty_check_classification,
+      "! <p class="shorttext">Additional Check Class</p>
+      "! Consistency Check Class
+      checkclass                TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Result Derivation Indicator</p>
+      "! Result Process Derived by BAdI
+      resultderivationindicator TYPE abap_bool,
+    END OF ty_attributes.
+
+  TYPES:
+    "! <p class="shorttext">Consistency Check ID</p>
+    "! Check ID (EDCK) v1
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
+      "! <p class="shorttext">Check ID Definition</p>
```

Comment:
Should the title here mention attributes as well?

Reply:
@GuilhermeSaraiva96 : Is it possible to do corrections in descriptions after the UX review. I am planning to do UX and UA reviews together, so that might lead to some title changes